### PR TITLE
Implement logging system in logger.py and integrate it in App.py

### PR DESCRIPTION
**Pull request by SatyaD012**
### Description
This pull request implements a logging system using a dedicated logger class in `logger.py`. The logging configuration includes:
- **Log file**: The default log file is `app.log` located in the root directory.
- **Log level**: The minimum logging level is set to `INFO`, with options for other levels such as WARNING and ERROR.

### Changes Made
- Created `logger.py` with logging configuration.
- Integrated logging into `App.py` to track application startup and shutdown events.
- Added a `logs` folder to store log files (added to `.gitignore` for version control).

### Screenshot
Attached is a screenshot of the log output demonstrating that the logging system is working correctly.

Screenshot 2024-10-05 191811

### CODE REVIEW

File: logger.py
Comment (yashasvini121) [inline logger.py:1]: You need to create a proper Logger Class as mentioned in the #53.

File: App.py
Comment (yashasvini121) [inline App.py:1]: It appears that all the code has been removed from `App.py`. Please restore the necessary code and also implement logging at relevant points.

**Review by yashasvini121 — CHANGES_REQUESTED** [review]
The altered files are showing up in binary format. Please address this issue to ensure that the files are in the correct format.

Additionally, it seems that several requirements outlined in issue #53 are missing. Please revisit the instructions and make sure to follow them thoroughly.

**Comment by yashasvini121** [comment]
@SatyaD012, updates?

**Comment by SatyaD012** [comment]
Hii 
I make some changes and  also added code to app.py .I will update it to you soon..

**Pull request closed without merging**